src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -189,8 +189,8 @@
   <data name="Place_open_brace_on_new_line_for_lambda_expression" xml:space="preserve">
     <value>Place open brace on new line for lambda expression</value>
   </data>
-  <data name="Place_open_brace_on_new_line_for_methods" xml:space="preserve">
-    <value>Place open brace on new line for methods</value>
+  <data name="Place_open_brace_on_new_line_for_methods_local_functions" xml:space="preserve">
+    <value>Place open brace on new line for methods and local functions</value>
```

Comment:
@nguerrera Are we safe to do this now for loc, or should we leave the resource unchanged for RTW, and update in the `post-dev15` branch after this merges there?